src/datepicker/day.js

```diff
@@ -101,15 +125,77 @@ export default class Day extends React.Component<DayPropsT, DayStateT> {
     );
   };
 
+  isSelected() {
+    if (this.props.isRange) {
+      return (
+        isSameDay(this.props.date, this.props.value[0]) ||
+        isSameDay(this.props.date, this.props.value[1])
+      );
+    } else {
+      return isSameDay(this.props.date, this.props.value);
+    }
+  }
+
+  // calculated for range case only
+  isPseudoSelected() {
+    const {date, value} = this.props;
+    if (value.length > 1) {
+      return isDayInRange(date, value[0], value[1]);
+    }
+  }
+
+  // calculated for range case only
+  isPseudoHighlighted() {
+    const {date, value, highlightedDate} = this.props;
+    if (highlightedDate && value[0] && !value[1]) {
+      if (isAfter(highlightedDate, value[0])) {
+        return isDayInRange(date, value[0], highlightedDate);
+      } else {
+        return isDayInRange(date, highlightedDate, value[0]);
+      }
+    }
+  }
+
   getSharedProps() {
+    const {date} = this.props;
+    const $isHighlighted = isSameDay(date, this.props.highlightedDate);
+    const $selected = this.isSelected(date);
+    const $hasRangeHighlighted =
+      this.props.isRange &&
+      this.props.value.length === 1 &&
```

Comment:
examples on netlify are breaking here or on L185
https://deploy-preview-708--baseui.netlify.com/?selectedKind=Unstable_Datepicker&selectedStory=Stateless%20datepicker&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

Reply:
fixed this one